src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+.firstSearch {
+  margin-top: 20px;
+}
+
+.searchBar {
+  box-sizing: border-box;
+  width: 100%;
+  color: #3d4e61;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bigSearch {
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  background-size: 19px 19px;
+  background-position: 25px 26px;
+}
+
+.bigSearch::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.bigSearch:hover {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigSearch:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.smallSearch {
+  height: 42px;
+  padding-left: 33px;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  background-size: 11px 11px;
+  background-position: 15px 16px;
+}
+
+.smallSearch::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.smallSearch:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.smallSearch:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+}
```

Comment:
you repeat yourself with this properties, create `:focus` styles for `searchBar` only
```suggestion
```

Reply:
done